code/modules/mob/living/carbon/monkey/diona.dm

```diff
@@ -163,13 +165,15 @@
 	if(donors.len == 5)
 		ready_evolve = 1
 		to_chat(src, "<span class='good'>You feel ready to move on to your next stage of growth.</span>")
-	else if(donors.len == 2)
-		to_chat(src, "<span class='good'>You feel your awareness expand, and realize you know how to understand the creatures around you.</span>")
 	else if(donors.len == 4)
```

Comment:
no it was like that
only riri can tell me to fix it at this point and I won't listen to him